aim/sdk/objects/image.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import os.path
 
-from PIL import Image as PILImage
+from PIL import Image as PILImage, ImageSequence
```

Comment:
can we import `ImageSequence` as `PILImageSequence`? I got confused for a moment, when looking at the code I thought it was `aim.Images` 😆

Reply:
yeah nice recommendation:)